:rotating_light: :rotating_light: Source Google Ads: update schemas and names of default streams (#31048)
airbytehq/airbyte: docs/integrations/sources/google-ads-migrations.md

# Google Ads Migration Guide

## Upgrading to 2.0.0

> This release includes enhancements to the default schemas and updates to stream names so user can easier match it to Google Ads resource. Users should refresh the source schema and reset affected streams after upgrading to ensure uninterrupted syncs.

Reviewer: ```suggestion
This release updates the Source Google Ads connector so that its default streams and stream names match the related resources in [Google Ads API](https://developers.google.com/google-ads/api/fields/v14/ad_group_ad).

Users should:
- Refresh the source schema
- And reset affected streams after upgrading to ensure uninterrupted syncs.
```
